example.py

```diff
@@ -0,0 +1,23 @@
+import numpy as np
```

Comment:
same here, can you put this file in `scripts_MATTER`?

Reply:
Moved!